feature-test/index.md

```diff
@@ -57,6 +57,36 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc aliquet ligula sed
 
 ## Code group
 
```

Comment:
Perhaps the header should not be `Code group`, but `Diff`.

Reply:
We'll extend this with a future update.